R/transfer.R

```diff
@@ -0,0 +1,37 @@
+#' Transferred assets among funding account and other accounts
+#'
+#' Exposes the \code{GET /sapi/v1/asset/transfer} endpoint.
+#'
+#' @inheritParams trade-parameters
+#' @inheritParams limit-5-30
+#' @param ... Additional parameters to pass they query
+#' @return A data frame.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' transferred_assets(type = "FUNDING_MAIN")
+#' }
+transferred_assets <- function(
```

Comment:
Please can you rename the function to `wallet_asset_transfer`. This is consistent with the existing naming convention which uses the section from https://binance-docs.github.io/apidocs/spot/en/ and then extracts the function name from the API endpoint.

Reply:
Makes sense. Changed on next commit